services/QuillLMS/app/models/sales_form_submission.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class SalesFormSubmission < ApplicationRecord
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :email, presence: true
+  validates :phone_number, presence: true
+  validates :zipcode, presence: true
+  validates :collection_type, presence: true
```

Comment:
Are the :collection_type values going to be fixed (e.g. selected from a drop down)?  If so, it might be worth adding that [restriction](https://github.com/empirical-org/Empirical-Core/blob/1bb95b65638d1c784cf2d23a7e074c11ff7fd99e/services/QuillLMS/app/models/plan.rb#L41) now.